src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -130,22 +128,48 @@ export class TechRecordSummaryComponent implements OnInit {
     this.store.dispatch(updateEditingTechRecord({ techRecord: this.vehicleTechRecordCalculated }));
   }
 
-  findAxleToRemove(axles: Axle[]): number {
-    let previousAxleRow = 1;
+  handleFormStateOriginal(event: any): void {
```

Comment:
Why are we leaving the original in place?

Reply:
My bad, debugging... Now removed 🙂